src/ig/IGExporter.ts

```diff
@@ -653,9 +653,15 @@ export class IGExporter {
       menuItem += `${prefixSpaces}</li>${EOL}`;
     } else {
       menuItem += `${prefixSpaces}<li>${EOL}${prefixSpaces}${'  '}`;
-      if (item.url) menuItem += `<a href="${item.url}">`;
+      if (item.url || item.openInNewTab) {
```

Comment:
Why `|| item.openInNewTab`?  If there is no URL, then what is the point of opening in a new tab?  What do browsers even do in that case? Open a blank page?

Reply:
I was thinking that since URL wasn't strictly required to be present, I would support the weird case for if there was a new-tab keyword and no URL. If there is no URL, the menu item isn't clickable and you get the text cursor (the one that looks like an "I"). Since that doesn't actually get you any extra functionality, I can remove the extra check for `item.openInNewTab` and make the code simpler. I think a related change should also be to only add the `external` image if there is a URL specified as well.